modules/json_content/src/Controller/FrontController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Drupal\json_content\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+class FrontController extends ControllerBase {
+
+  /**
+   * Displays the login form on the homepage and redirects authenticated users.
+   */
+  public function frontpage() {
+    $build = [];
+    if (\Drupal::currentUser()->isAnonymous()) {
+      $build['heading'] = [
+        '#type' => 'markup',
+        '#markup' => $this->t('Please login for access to the content repository.'),
```

Comment:
Hm. The login form is always shown right below this text. And we want people to use that login form because it redirects to the homepage which will redirect people to the proper place.